pynicotine/gtkgui/ui/settings/icons.ui

```diff
@@ -455,7 +455,7 @@
                   <object class="GtkLabel">
                     <property name="visible">True</property>
                     <property name="can_focus">False</property>
-                    <property name="label" translatable="yes">Hilite:</property>
+                    <property name="label" translatable="yes">Nick Hilite:</property>
```

Comment:
This label is incorrect. The icon is also used whenever a new message is sent in a chat room.

Reply:
i revert the label to'Hilite" as it was before sorry for that ...